Doc/tools/templates/download.html

```diff
@@ -31,8 +31,7 @@ <h1>{% trans %}Download Python {{ dl_version }} documentation{% endtrans %}</h1>
 
 {% if last_updated %}<p><b>{% trans %}Last updated on: {{ last_updated }}.{% endtrans %}</b></p>{% endif %}
 
-<p>{% trans %}To download an archive containing all the documents for this version of
-Python in one of various formats, follow one of links in this table.{% endtrans %}</p>
+<p>{% trans %}+Download an archive containing all the documentation for this version of Python:{% endtrans %}</p>
```

Comment:
Oops, we wanted this instead:

```suggestion
<p>{% trans %}Download an archive containing all the documentation for this version of Python:{% endtrans %}</p>
```